fastai/tabular/learner.py

```diff
@@ -1,17 +1,21 @@
-# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/43_tabular.learner.ipynb (unless otherwise specified).
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/nbs/43_tabular.learner.ipynb (unless otherwise specified).
 
 __all__ = ['TabularLearner', 'tabular_learner']
 
 # Cell
 from ..basics import *
 from .core import *
 from .model import *
+from .data import *
 
 # Cell
 class TabularLearner(Learner):
     "`Learner` for tabular data"
-    def predict(self, row):
-        "Predict on a Pandas Series"
+    def predict(
```

Comment:
We have decided to use a format when docmenting `self` that keeps self on the same line as the function name.  So in this case, you would combine `def predict(self,` onto a single line

Reply:
Noted. Thanks

Reply:
Still awaiting this change @moarshy

Reply:
is it ok to close this PR and open a new one? I hv tried few things to roll back on the mistakes but maybe I might have done some damage.